### DISCUSSION TITLE
new mikrotik feature, allow to synchronize only one RB and not the whole system (RB)

### DISCUSSION THREAD

**Discussion by carlos0424** [hotspotbilling/phpnuxbill#127] [Ideas]
is it possible to add a function to be able to synchronize only one Mikrotik RB to update PPPOE profiles and so on, since if you have 7 Mikrotik you have to wait until all RBs are swept, you should be able to filter and allow to synchronize only one Mikrotik RB.

**Comment by Focuslinkstech** [comment]
Noted! it will be considered in the future updates
don't forget to test the radius features and also give feedback
